ci/install-aws.inc.sh

```diff
@@ -19,4 +19,6 @@ aws configure set s3.signature_version s3v4
 # or whenever the deb packages have been upgraded to awscli version 2.0.
 # exe_and_grep_q "aws --version | head -1" "^aws-cli/2\\."
 exe_and_grep_q "aws --version | head -1" "^aws-cli/[12]\\."
+# check that we get a recent enough aws client.
+exe_and_grep_q "aws cloudformation detect-stack-drift help | grep -o DRIFT | sort -u" "DRIFT"
```

Comment:
I guess this is enough;
`exe_and_grep_q "aws cloudformation detect-stack-drift help" DRIFT